### PR TITLE
recommendation: change filter banner from a notification

### PR DESCRIPTION
**Pull request by kiley-mitti**
There's not really an official recommendation here, and Notification Banner may well be the way to go here, but I made a small stylistic change here to make the filter notification fall more inline with the way rux-log does it. Absolutely feel free to deny this.

### CODE REVIEW

File: src/Components/Alerts/Alerts.tsx [hunk 1]
```diff
@@ -72,7 +71,7 @@ const Alerts = () => {
           </RuxSelect>
         </div>
       </div>
-      <RuxNotification open={openBanner} small hide-close>
+      <div className="filter-notification" hidden={!dataFiltered}>
```
[the remark this answers is not in the document]
Comment (kiley-mitti) [reply src/Components/Alerts/Alerts.tsx:74]: Into it! One moment and I'll get that in.

[the remark this answers is not in the document]
Comment (kiley-mitti) [reply src/Components/Alerts/Alerts.tsx]: That'll. show up next build :) thanks for the idea! :)

File: src/Components/Alerts/Alerts.css [hunk 2]
```diff
@@ -24,6 +24,18 @@
...
 
+.filter-notification {
+  background-color: var(--color-background-surface-default);
+  padding: var(--spacing-2);
+  overflow-x: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  & rux-button::part(container):hover {
+    box-shadow: none;
```
Comment (FMorrison87) [inline src/Components/Alerts/Alerts.css:35]: Can you add in padding-inline: var(--spacing-1) so it doesn't have so much space between the text?

**Review by FMorrison87 — CHANGES_REQUESTED** [review]

File: src/Components/Alerts/Alerts.css (hunk 2 above)
Comment (kiley-mitti) [reply src/Components/Alerts/Alerts.css:36]: sure thing! will be in the next deploy preview

**Review by NicDominguez — COMMENTED** [review]
I think we should get a design decision on this before merging as we use rux-notification for many tables in dashboard and this pattern would change those as well. @kiley-mitti will you check in with design about making this change?

File: src/Components/Alerts/Alerts.css (hunk 2 above)
Comment (FMorrison87) [reply src/Components/Alerts/Alerts.css:36]: *Chef's kiss*

**Comment by kiley-mitti** [comment]
I'm on it! Be back soon with more info. :)

**Comment by kiley-mitti** [comment]
@NicDominguez I asked in AstroInternal and the main consensus seems to be that we don't have a strong pattern for announcing filtering but the simpler display is preferred to distinguish it from Notification Banner.

**Review by Paterma — APPROVED** [review]

**Review by NicDominguez — APPROVED** [review]

**Pull request merged**